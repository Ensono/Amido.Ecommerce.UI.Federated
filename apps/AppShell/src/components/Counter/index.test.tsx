import {render, waitFor} from '@testing-library/react'

import {Counter} from '.'

describe('Counter', () => {
  it('matches the snapshot', async () => {
    const {container} = render(<Counter count={0} increment={jest.fn()} reset={jest.fn()} />)
    await waitFor(() => {
      expect(container).toMatchSnapshot()
    })
  })
})
