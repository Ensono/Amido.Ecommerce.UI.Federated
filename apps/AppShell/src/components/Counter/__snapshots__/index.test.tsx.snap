// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter matches the snapshot 1`] = `
<div>
  <div>
    <h2
      data-testid="count"
    >
      Count: 
      <span>
        0
      </span>
    </h2>
    <button
      data-testid="increment-button"
    >
      Make number go up plz
    </button>
    <button
      data-testid="reset-button"
    >
      Reset
    </button>
  </div>
</div>
`;
